ChangeLog

```diff
@@ -7,6 +7,10 @@ Bugfix
      invalidated keys of a lifetime of less than a 1s. Fixes #1968.
    * Fix failure in hmac_drbg in the benchmark sample application, when
      MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
+   * Fix compilation failure for configurations which use compile time
+     replacements of standard calloc/free functions through the macros
+     MBEDTLS_PLATFORM_CALLOC_MACRO and MBEDTLS_PLATFORM_FREE_MACRO.
+     Fixes #1642.
```

Comment:
Since the first issue that this PR fixes is #882 , we should change the log, and credit the OP

Reply:
Thanks, will do that - however, shouldn't we also credit the other independent reporters of the issue?

Reply:
As long as it's mpt a long list, I don't really mind, but basically, we should credit the first issue reporter, as I see it

Reply:
Fixed.